app/src/main/res/values/strings.xml

```diff
@@ -151,6 +151,10 @@
     <string name="follow">Nachstieg</string>
     <string name="colorize_tourbook_entries">Tourenbucheinträge färben</string>
 
+    <string name="reset_database">Auf Werkseinstellungen zurücksetzen</string>
+    <string name="reset_database_confirm">Datenbank (inkl. Tourenbuch) löschen. Fortfahren?</string>
```

Comment:
Better: `Dies löscht die gesamte Datenbank, inkl. Tourenbuch. Fortfahren?`

Reply:
Thanks, I changed it.